### DISCUSSION TITLE
Monero / Zephyr protocol / Tabo protocol - crypto trading

### DISCUSSION THREAD

**Discussion by undergroundBlock** [P0nk/Cosmic#246] [Ideas]
Hi, it would be very nice and very attractive for more types of population in the internet market to add trading with private cryptocurrencies like Tabo protocol and Monero XMR or Zephyr protocol in marketplace, you think it's possible and can be contribute by community and can be added to cosmic repository?

I think it will get a lot of attention and make Cosmic repository
 cosmically hyped

**Comment by P0nk** [comment]
No
